### DISCUSSION TITLE
Installation 2.0.3

### DISCUSSION THREAD

**Discussion by 918483** [savonet/liquidsoap#2362] [User Support]
Could you tell me the correct way to install liquidsoap to use mp3 and acc in the transmission and have ffmpeg already included for relay because I've been doing installations for 3 days and in the ffmpeg part I always get an error when compiling the 

ERROR while compiling ffmpeg-avutil.1.1.4
ERROR while compiling ocurl.0.9.2

I'm using centos 7 + icecast + shoutcast

**Comment by toots** [comment]
If this is the same issues as in https://github.com/savonet/liquidsoap/discussions/2354, you should make sure that you have uninstalled ffmpeg from centos and use the one compiled locally.

**Reply by 918483** [reply]
could you tell me how to compile it locally?
because I have already formatted the machine more than 8x and every time I install the error even without installing ffmpeg if you can give me a step by step installation I appreciate the help.

**Reply by 918483** [reply]
https://www.elephdev.com/index.php/cphp/317.html?ref=addtabs&lang=en
